### PR TITLE
Fix/phpstan level 0

### PR DESCRIPTION
**Pull request by lucatume**
The scope of this PR is to update/fix/refactor wp-browser code to pass w/o any error phpstan checks of level 0.

Due to wp-browser requirement of back-compatibility and support for Codeception 2.5+ 
 use `_build/vendor_prepare 5.6 4.0` to pull the correct dependencies and ensure the checks are correct against latest version of Codeception 4.0+.

To test locally:

* `_build/vendor_prepare 5.6 .4.0`
* `make phpstan`

### CODE REVIEW

**Comment by lucatume** [comment]
@szepeviktor this is the PR I mentioned. If you're still of that mind, I could use the help here and make this pass on builds that use Codeception 4.0.

It would be nice to do the same for other versions of Codeception too, but I'm going for something that's attainable here.

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -22,6 +22,15 @@ jobs:
       - name: Sniff files
         run: make sniff
 
+  phpstsan:
+    name: Run phpstan on source files
+    runs-on: ubuntu-18.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: phpstan
```
Comment (szepeviktor) [inline .github/workflows/ci.yaml:31]: Could you write PHPStan or Static analysis here?

File: src/tad/WPBrowser/patchwork.php [hunk 2]
```diff
@@ -27,7 +27,6 @@ function includePatchwork()
 function configurePatchwork($config)
 {
     includePatchwork();
-    ;
```
Comment (szepeviktor) [inline src/tad/WPBrowser/patchwork.php:30]: What a catch!!! :)

**Comment by szepeviktor** [comment]
> [ERROR] Found 24 errors     

Glad to see number of problems approaching Zero!

**Comment by szepeviktor** [comment]
Objective complete 🚢

**Comment by szepeviktor** [comment]
You may start Level 2 analysis :)

File: tests/_support/_generated/UnitTesterActions.php [hunk 3]
```diff
@@ -1,4 +1,8 @@
+<<<<<<< HEAD
+<?php  //[STAMP] c82f7589be5dd84bad73693ed18c0018
+=======
 <?php  //[STAMP] dd083e05f8f2ff542d59f572e7b0c030
+>>>>>>> master
```
Comment (szepeviktor) [inline tests/_support/_generated/UnitTesterActions.php:5]: @lucatume Attention!

Committed conflict.

Comment (szepeviktor) [reply tests/_support/_generated/UnitTesterActions.php:5]: From https://github.com/lucatume/wp-browser/commit/deb1203d61db0116c5cb5b9ea5ca6293ce00c635#diff-ea1bf73795500649ccdc0b85827b04ce

**Comment by szepeviktor** [comment]
# 🎉

**Pull request merged**